app/decorators/estimate_decorator.rb

```diff
@@ -24,7 +24,7 @@ class << self
     def cost_options_json
       cost_types = COST_TYPE_TO_LABEL.sort_by { |_key, value| value }
       strings = cost_types.map do |key, value|
-        "{label: '#{value}', value: '#{key}'}"
+        "{label: '#{value.humanize}', value: '#{key}'}"
```

Comment:
@axamei not sure why this needed ? The values in the table are already humanize in COST_TYPE_TO_LABEL

Reply:
Without this, the values in the dropdown were lowercase. See: https://percy.io/Princeton-University-Library/approvals/builds/3967987

Reply:
Is there a time we would want them to be lower case.  We could just change the values here to be cased correctly: https://github.com/pulibrary/approvals/pull/516/files#diff-01b759627e814d50d18b24e8e2354637R37

Reply:
@carolyncole I don't think there would be a need for them to be displayed in lower case. Changing the values to have correct capitalization where you pointed makes sense. Did you want me to revert the last commit and change the values in the location you suggested?